popup.html

```diff
@@ -0,0 +1,33 @@
+<style>
+html {
+    background-color: #232628;
+    color: #CCCCCC;
```

Comment:
I can look into that.  I think you may have to handle that logic yourself.  I'll add that as an issue to follow up on.